src/Cli/dotnet/Commands/Run/Api/RunApiCommand.cs

```diff
@@ -82,6 +82,13 @@ internal abstract class RunApiOutput
 {
     private RunApiOutput() { }
 
+    /// <summary>
+    /// When the API shape or behavior changes, this should be incremented so the callers (IDEs) can act accordingly
+    /// (e.g., show an error message when an incompatible SDK version is being used).
+    /// </summary>
+    [JsonPropertyOrder(-1)]
+    public int Version { get; } = 1;
```

Comment:
It might also make sense for caller to pass in the "version" of the API they are using, implying the set of behaviors they expect, and the run-api command can reject the request in a well-defined way if it's not "compatible" with that version.

Reply:
Actually, it looks like this property is on the base type? So the caller can always just read the version that comes out. It would be good to verify that this property is present in an error case.